include/fe/fe_abstract.h

```diff
@@ -150,9 +150,17 @@ class FEAbstract : public ReferenceCountedObject<FEAbstract>
                        const std::vector<Point> * const pts = nullptr,
                        const std::vector<Real> * const weights = nullptr) = 0;
 
-  virtual void reinit_dual_shape_coeffs (const Elem * elem,
-                                         const std::vector<Point> & pts,
-                                         const std::vector<Real> & JxW) = 0;
+  /**
+   * This re-computes the dual shape function coefficients.
+   * The dual shape coefficients are utilized when calculating dual shape functions.
+   * This does not support InfFE type.
+   */
+  virtual void reinit_dual_shape_coeffs (const Elem * /*elem*/,
+                                         const std::vector<Point> & /*pts*/,
+                                         const std::vector<Real> & /*JxW*/)
+  {
+    libmesh_error_msg("Dual shape coefficient calculation has not been implemented for this FE type.");
```

Comment:
Likewise with a fixup for this commit, if I'm understanding correctly and it broke --enable-ifem builds.